odoo/http.py

```diff
@@ -862,7 +862,6 @@ def not_found(self, description=None):
 #----------------------------------------------------------
 # Controller and route registration
 #----------------------------------------------------------
-addons_manifest = {}
```

Comment:
The one thing which would actually be useful for backwards compatibility would be to re-export `odoo.addons.modules.module.addons_manifest` from `http`.